app/lib/email_sender.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter_email_sender/flutter_email_sender.dart';
+import 'package:flutter/material.dart';
+
+class EmailSender {
+  Future<void> send(
+      String emailBody, String eSubject, String eRecipients) async {
+    final Email email = Email(
+      body: emailBody,
+      subject: eSubject,
+      recipients: [eRecipients],
+      attachmentPaths: null,
+      isHTML: false,
+    );
+    String emailResponse;
+    try {
+      await FlutterEmailSender.send(email);
+      emailResponse = 'success';
+    } catch (error) {
+      emailResponse = error.toString();
+    }
+  }
+}
+
+class EmailRoute extends StatelessWidget {
+  final formKey = GlobalKey<FormState>();
+
+  final email = EmailSender();
+  final recipientController = TextEditingController();
+  final subjectController = TextEditingController();
+  final bodyController = TextEditingController();
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Email Form'),
+        ),
+        body: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Form(
+              key: formKey,
+              child: Column(children: <Widget>[
+                TextFormField(
+                  decoration: InputDecoration(hintText: 'Email'),
+                  controller: recipientController,
+                  validator: (value) {
+                    if (value.isEmpty) {
+                      return 'Please enter email';
+                    } else
+                      return null;
+                  },
+                ),
+                TextFormField(
+                  decoration: InputDecoration(hintText: 'Email Subject'),
+                  controller: subjectController,
+                ),
+                TextFormField(
+                  decoration: InputDecoration(hintText: 'Message'),
+                  controller: bodyController,
+                ),
+                RaisedButton(
+                  onPressed: () {
+                    if (formKey.currentState.validate()) {
+                      print('valid');
+                      email.send(bodyController.text, subjectController.text,
+                          recipientController.text);
+                    }
+                  },
+                  child: Text('Submit'),
```

Comment:
Once we started real development (no need to change rn), as for best practice, we should create custom widgets so we can use it anywhere on the app. We should make things reusable.

Reply:
Alright!